src/main/java/org/folio/fqm/resource/MigrationController.java

```diff
@@ -15,4 +19,24 @@ public class MigrationController implements FqmVersionApi {
   public ResponseEntity<String> getFqmVersion() {
     return new ResponseEntity<>(migrationService.getLatestVersion(), HttpStatus.OK);
   }
+
+  @Override
+  public ResponseEntity<FqmMigrateResponse> fqmMigrate(FqmMigrateRequest fqmMigrateRequest) {
+    MigratableQueryInformation migratableQueryInformation = new MigratableQueryInformation(
+      fqmMigrateRequest.getEntityTypeId(),
+      fqmMigrateRequest.getFqlQuery(),
+      fqmMigrateRequest.getFields()
+    );
+
+    migrationService.migrate(migratableQueryInformation);
+
+    FqmMigrateResponse fqmMigrateResponse = new FqmMigrateResponse()
+      .entityTypeId(migratableQueryInformation.entityTypeId())
+      .fqlQuery(migratableQueryInformation.fqlQuery())
+      .fields(migratableQueryInformation.fields())
+      .warnings(migratableQueryInformation.warnings().stream()
```

Comment:
I'd like to return the type and description in the API response, so the description can be returned to the user and the type can be used by the receiving module/etc to handle the case appropriately